Documentation/coding-guidelines/adding-api-guidelines.md

```diff
@@ -0,0 +1,124 @@
+#Add APIs
```

Comment:
Suggestion: link to the important pre-reading here which seems to include

https://github.com/dotnet/corefx/blob/master/Documentation/architecture/net-platform-standard.md
https://github.com/dotnet/corefx/blob/master/Documentation/coding-guidelines/project-guidelines.md